Source/DafnyLanguageServer.Test/Synchronization/CloseDocumentTest.cs

```diff
@@ -34,7 +35,14 @@ public class CloseDocumentTest : DafnyLanguageServerTestBase, IAsyncLifetime {
       var documentItem = CreateTestDocument(source);
       await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
       await CloseDocumentAndWaitAsync(documentItem);
-      Assert.Null(await Projects.GetResolvedDocumentAsync(documentItem.Uri));
+      for (int attempt = 0; attempt < 5; attempt++) {
```

Comment:
nit: 500ms seems like a long time to allow

Reply:
Does it matter? Some test timeouts are >60s

Reply:
I don't think it matters for the sake of CI, but I think setting too high an allowance could matter for catching performance regressions. Not a blocker though.